src/ansys/fluent/core/session_pure_meshing.py

```diff
@@ -111,6 +111,16 @@ def two_dimensional_meshing(self):
         self._base_meshing.two_dimensional_meshing_workflow.reinitialize()
         return self._base_meshing.two_dimensional_meshing_workflow
 
+    def load_workflow(self, file_path: str):
+        """Load a saved workflow."""
+        self._base_meshing.load_workflow(file_path=file_path).load()
+        return self._base_meshing.load_workflow(file_path=file_path)
+
+    def create_workflow(self):
+        """Create a new meshing workflow."""
```

Comment:
```suggestion
        """Create a meshing workflow."""
```
The "new" is unnecessary as you can't create an "old" workflow.